projects/packages/my-jetpack/_inc/components/onboarding-tour/index.tsx

```diff
@@ -73,41 +73,17 @@ const OnboardingTour: FC< { open?: boolean } > = ( { open = true } ) => {
 							content: (
 								<>
 									<h1 className="myjetpack-onboarding-welcome-tour__heading">
-										{ __( 'Making your site super fast', 'jetpack-my-jetpack' ) }
+										{ __( 'Make your site super fast', 'jetpack-my-jetpack' ) }
 									</h1>
 									<p className="myjetpack-onboarding-welcome-tour__text">
 										{ __(
-											'We’re already checking how your site stacks up and will suggest speed optimizations in a few minutes.',
+											'Make your site faster and more user-friendly by boosting loading speeds and SEO in minutes.',
 											'jetpack-my-jetpack'
 										) }
 									</p>
 								</>
 							),
 						},
-						// {
-						// 	image: (
-						// 		<WelcomeTourImage
-						// 			className="myjetpack-onboarding-welcome-tour__image"
-						// 			nonAnimatedSrc={ TourImage3 }
-						// 			nonAnimatedSrc2x={ TourImage3x2 }
-						// 			animatedSrc={ TourImage3 }
-						// 			animatedSrc2x={ TourImage3x2 }
-						// 		/>
-						// 	),
-						// 	content: (
-						// 		<>
-						// 			<h1 className="myjetpack-onboarding-welcome-tour__heading">
-						// 				{ __( 'Your entire site, safe in the cloud', 'jetpack-my-jetpack' ) }
-						// 			</h1>
-						// 			<p className="myjetpack-onboarding-welcome-tour__text">
-						// 				{ __(
-						// 					'We’ve started to back up your entire site in the cloud. Restore with one click if anything goes wrong.',
-						// 					'jetpack-my-jetpack'
-						// 				) }
-						// 			</p>
-						// 		</>
-						// 	),
-						// },
```

Comment:
Should we clean this up?

Reply:
yes, this slide will have to be changed if we put it back in so it doesn't make sense to keep it in the code